pyeudiw/x509/chain_builder.py

```diff
@@ -85,43 +85,21 @@ def gen_certificate(
             )
         ]
 
-        if organization_identifier:
+        subject_names = x509.Name(x5c_names)
+
+        if org_name:
             x5c_names.append(
-                x509.NameAttribute(NameOID.ORGANIZATION_IDENTIFIER, organization_identifier)
+                x509.NameAttribute(NameOID.ORGANIZATION_IDENTIFIER, org_name)
```

Comment:
the organization identifier is an unique code issued within a specific onboardign system, it doesn't match with the org name